doc/release/multilocale.rst

```diff
@@ -4,233 +4,196 @@
-           export GASNET_CSPAWN_CMD="srun -N%N %C"
-
-
-      #. To simulate multiple Chapel locales with one workstation,
-         you can request that GASNet spawn its child functions on
-         your local machine using:
-
-         .. code-block:: bash
-
-           export GASNET_SPAWNFN=L
-
-         If you have problems with the local spawn, you may need to
-         set ``GASNET_MASTERIP`` to your IP address.
-
-
-      #. For other options, please refer to:
-
-         ``$CHPL_HOME/third-party/gasnet/gasnet-src/udp-conduit/README``
-
-   #. If you are using GASNet's ibv conduit, configure the environment to
-      launch program instances.  We have experience with these configurations:
-
-      i. For clusters using SLURM, enable slurm-gasnetrun_ibv:
-
-         .. code-block:: bash
-
-           export CHPL_LAUNCHER=slurm-gasnetrun_ibv
-
-         See :ref:`readme-launcher` for other options available, such
-         as setting the time limit or selecting the type of node.
+   based on these settings.
 
+#. Compile your Chapel program as usual:
 
-      #. To launch InfiniBand jobs with SSH, use the following
+   .. code-block:: bash
 
-         .. code-block:: bash
+     chpl -o hello6-taskpar-dist $CHPL_HOME/examples/hello6-taskpar-dist.chpl
 
-           export CHPL_LAUNCHER=gasnetrun_ibv
-           export GASNET_SSH_SERVERS="host1 host2 host3 ..."
-           export GASNET_IBV_SPAWNER=ssh
+#. Set any environment variables necessary to control the launcher.
+   See :ref:`readme-launcher` or documentation for your platform.
+   For example, to specify which servers the UDP conduit should use as
+   described in :ref:`using-udp`:
 
+   .. code-block:: bash
 
-      #. We've observed job launch hangs in some systems with InfiniBand
-         that were resolved by limiting the memory available for
-         communication, for example with:
+     export GASNET_SPAWNFN=S
+     export GASNET_SSH_SERVERS="host1 host2 host3 ..."
 
-         .. code-block:: bash
+#. Specify the number of locales on the command line. For example:
 
-           export GASNET_PHYSMEM_MAX=1G
+   .. code-block:: bash
 
+     ./hello6-taskpar-dist -nl 2
 
-      #. For more information on these and other available GASNet options
-         other options, including configuring to launch through MPI,
-         please refer to:
+   runs our Hello World program on 2 locales.
+
+What is this _real program?
++++++++++++++++++++++++++++
+
+When you compile a Chapel program for multiple locales, you should
+typically see two binaries (e.g., hello6-taskpar-dist and
+hello6-taskpar-dist_real).  The first binary contains code to
+launch the Chapel program onto the compute nodes. The second contains
+the program code itself.
+
+
+What is GASNet?
++++++++++++++++
+
+.. _what-is-gasnet:
+
+
+GASNet is a one-sided communication and active message library being
+developed by Lawrence Berkeley National Lab and UC Berkeley.  For details,
+refer to:
```

Comment:
refer to \`GASNet's official documentation <http://gasnet.cs.berkeley.edu>`_

(we don't want to point people to [third-party/gasnet/README](https://github.com/chapel-lang/chapel/blob/master/third-party/README), which is our modified version with a bunch of licensing stuff)